libs/wcm/hooks/useSession.js

```diff
@@ -119,7 +119,7 @@ export const useSession = () => {
   );
 
   useEffect(() => {
-    if (signClient?.session && !hasLoaded) {
+    if (signClient?.session && !hasLoaded && isEnabled) {
```

Comment:
Shouldnt we loadSessions if isEnabled become enabled? Dont we need it in the dep array in that case?